python/iceberg/api/expressions/literals.py

```diff
@@ -35,8 +35,8 @@ class Literals(object):
     EPOCH = datetime.datetime.utcfromtimestamp(0)
     EPOCH_DAY = EPOCH.date()
 
-    @staticmethod  # noqa: C901
-    def from_(value):
+    @staticmethod
+    def from_(value):  # noqa: C901
```

Comment:
Are these changes needed?

Reply:
yeah, they were one of the causes of the failures